src/tools/piv-tool.c

```diff
@@ -494,10 +494,7 @@ int main(int argc, char * const argv[])
 			action_count++;
 			break;
 		case 'Z':
-			/*
-			-Z is the same as -C but -Z says the cert is compressed.
-			https://github.com/OpenSC/OpenSC/commit/3a92bf7af5aaa85fdbc0434d0a55393e68e0f98d#commitcomment-9636487
-			*/
+			/* fall through */
```

Comment:
OK, I can move it down a line, but may be OK were it is.
 
http://security.coverity.com/blog/2013/Sep/gimme-a-break.html
says: "Comment heuristic"
"We also suppress a case label if there is a comment that matches [^#]fall.?thro?u, even if it's not on the last line."